app/frontend/src/aquifers/components/Search.vue

```diff
@@ -18,7 +18,7 @@
     <b-alert
         show
         variant="info"
-        class="container mb-3"
+        class="container mb-3 col-sm-12"
```

Comment:
```suggestion
        class="mb-3"
```

There's already a container wrapping this component